cmd/mockrecord.go

```diff
@@ -65,22 +65,27 @@ func (mr *MockRecord) GetCmd() *cobra.Command {
 				return
 			}
 
-
+			disableTele, err := cmd.Flags().GetBool("disableTele")
+			if err != nil {
+				mr.logger.Error(Emoji + "failed to read the disableTele flag")
+				return
+			}
 
 			proxyPort, err := cmd.Flags().GetUint32("proxyport")
 			if err != nil {
 				mr.logger.Error(Emoji + "failed to read the proxy port")
 				return
 			}
 
-			mr.mockRecorder.MockRecord(path,proxyPort, pid, dir)
+			mr.mockRecorder.MockRecord(path,proxyPort, pid, dir, disableTele)
 		},
 	}
 
 	serveCmd.Flags().Uint32("pid", 0, "Process id of your application.")
 	serveCmd.MarkFlagRequired("pid")
 
 	serveCmd.Flags().Uint32("proxyport", 0, "Choose a port to run Keploy Proxy.")
+	serveCmd.Flags().Bool("disableTele", false, "Switch for telemetry" )
```

Comment:
Make sure this flag doesn't show up in keploy help or any other command eaxmple or explaination.